### PR TITLE
remove default value + apply missing field on environment Layer

### PR DESCRIPTION
**Pull request by SebastienPeillet**
@pblottiere

### CODE REVIEW

File: src/core/database.py [hunk 1]
```diff
@@ -104,7 +104,8 @@ def _createFieldsForEnvironmentTable(self) -> QgsFields:
         fields.append(self._createFieldShortText("glareSever"))
         fields.append(QgsField("cloudCover", QVariant.Int))
         fields.append(QgsField("visibility", QVariant.Int))
-        fields.append(QgsField("subjective", QVariant.String, len=1))
+        fields.append(QgsField("subjectiveMam", QVariant.String, len=2))
+        fields.append(QgsField("subjectiveBirds", QVariant.String, len=2))
```
Comment (SebastienPeillet) [inline src/core/database.py:108]: @pblottiere subjective field changed into two new fields that has been added in last excel file

File: src/core/layers/sightings.py [hunk 2]
```diff
@@ -175,7 +167,9 @@ def _init_widgets(self, layer: QgsVectorLayer) -> None:
         ]
         setup = QgsEditorWidgetSetup("ValueMap", cfg)
         layer.setEditorWidgetSetup(idx, setup)
-        layer.setDefaultValueDefinition(idx, QgsDefaultValue("'foraging'"))
+        layer.setDefaultValueDefinition(
+            idx, QgsDefaultValue("{2839923C-8B7D-419E-B84B-CA2FE9B80EC7}")
```
Comment (pblottiere) [inline src/core/layers/sightings.py:171]: Could you defined this string `"{2839923C-8B7D-419E-B84B-CA2FE9B80EC7}"` in a dedicated global variable please?

File: src/core/database.py (hunk 1 above)
Comment (pblottiere) [reply src/core/database.py:108]: Ok :+1:

File: src/core/layers/sightings.py (hunk 2 above)
Comment (SebastienPeillet) [reply src/core/layers/sightings.py]: :+1: It's done

**Comment by pblottiere** [comment]
Thanks :+1:

**Pull request merged**